.github/scripts/test_and_publish.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+
+import subprocess, sys, os, tempfile
+from distantrs import Invocation
+from glob import glob
+
+INVOCATION_DATA_PATH = '/tmp/distant-rs-invocation.txt'
+
+# required to find listener.py
+this_path = os.path.abspath(os.path.dirname(__file__))
+test_script_path = f"{os.environ.get('GITHUB_WORKSPACE')}/renode/test.sh"
+inv_data_str = ""
+robot_name = glob("*.robot")[0].split(".")[0]
```

Comment:
these are actually provided as args, I think. Please verify

Reply:
You are right. I modified the script so that it only uses `sys.argv[1]` and no glob (it's not needed; shell expands the pattern).